libs/shared/ui-components/src/ColumnArticle/ColumnArticle.tsx

```diff
@@ -25,7 +25,7 @@ export const ColumnArticle: FC<TColumnArticleProps> = ({
           <ColumnArticleHeader header={header} level={2} />
         </header>
       )}
-      <main className="md:flex md:flex-wrap md:justify-between">
+      <div className="md:flex md:flex-wrap md:justify-between">
```

Comment:
There should only be [one `<main>` per page](https://html.spec.whatwg.org/multipage/grouping-content.html#the-main-element).